examples/common.h

```diff
@@ -39,7 +39,7 @@ struct gpt_params {
     float   mirostat_tau      = 5.00f; // target entropy
     float   mirostat_eta      = 0.10f; // learning rate
 
-    std::string model  = "models/lamma-7B/ggml-model.bin"; // model path
+    std::string model  = "models/llama-7B/ggml-model.bin"; // model path
```

Comment:
Readme mentions `./models/7B/ggml-model-q4_0.bin` so I think we should use this instead to unify these two.

```suggestion
    std::string model  = "models/7B/ggml-model-q4_0.bin"; // model path
```

Reply:
I prefer to keep 7B because that's how the directory is named when you download the model.

Also the instructiond should work no matter what model is located in 7B directory (Alpaca, Vicuna, etc.)

Reply:
I've implemented your suggested changes. The default model path is now defined in ``examples/common.h`` and this is used elsewhere implicitly, instead of repeating the string in several places.